nixnet/_session/base.py

```diff
@@ -335,6 +335,37 @@ def disconnect_terminals(self, source, destination):
         """
         _funcs.nx_disconnect_terminals(self._handle, source, destination)
 
+    def write_state_lin_schedule_change(self, state_value):
+        # type: () -> None
+        """Writes communication states of an XNET session.
+
+        This function writes a request for the LIN interface to change
+        the running schedule.
+
+        According to the LIN protocol, only the master executes schedules,
+        not slaves. If the
+        :any:`nixnet._session.intf.Interface.lin_master` property is false (slave),
+        this write function implicitly sets that property to true (master). If the
+        interface currently is running as a slave, this write returns an error,
+        because it cannot change to master while running.
+
+        Args:
+            state_value(str): Writes the desired state.
+        """
+        _funcs.nx_write_state(self._handle, constants.WriteState.LIN_SCHEDULE_CHANGE, state_value)
+
+    def write_state_lin_diagnostic_schedule_change(self, state_value):
```

Comment:
Like the other functions, I don't think we need "write_state" in the name.

So the question is what is a name that makes sense for just this function?

Would `change_lin_diagnostic_schedule` work?  My main concern is that is a similar name to `change_lin_schedule` but they take completely different types of parameters.